src/test/groovy/com/jvm_bloggers/utils/HtmlEmptinessCheckerSpec.groovy

```diff
@@ -0,0 +1,32 @@
+package com.jvm_bloggers.utils
+
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class HtmlEmptinessCheckerSpec extends Specification {
+
+    @Unroll
+    def "Should properly check for section emptiness when content = `#content`"() {
```

Comment:
Ok, I will add them but I am not sure if they give any value (especially when you know that `isEmpty()` is simply a negated `isNotEmpty()` call)